docker/colony-cdapp-dev-env-network

```diff
@@ -27,6 +27,9 @@ RUN git fetch origin $NETWORK_HASH --depth 1
 RUN git checkout $NETWORK_HASH
 RUN git submodule update --init --recursive --depth 1
 
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash && \
+    echo "source $HOME/.nvm/nvm.sh" >> $HOME/.bashrc
```

Comment:
i will actually check it out if it may just work without this, it's just sourcing nvm into bashrc so it can be used from bash without needing to reload the shell, but the shell may be reloaded in any case :thinking: